solutions/swb-reference/integration-tests/config/testEnv.yaml

```diff
@@ -15,6 +15,15 @@ terminatedEnvId: 'env-b44262fb-ed34-47ae-a4a4-d2d9d9c7a5d5' # id of an environme
 rootUserNameParamStorePath: '/swb/testEnv/rootUser/email'
 rootPasswordParamStorePath: '/swb/testEnv/rootUser/password'
 
+projectAdmin1UserNameParamStorePath: '/swb/testEnv/pa1/email'
```

Comment:
Make sure the `example.yaml` also has these values commented out. That file is used to tell users what _should_ be set up to run integ tests while this is the GH Actions stage file.